pkg/deploy/server/deployment_che.go

```diff
@@ -406,3 +411,48 @@ func GetFullCheServerImageLink(checluster *orgv1.CheCluster) string {
 	imageParts := strings.Split(defaultCheServerImage, separator)
 	return imageParts[0] + ":" + checluster.Spec.Server.CheImageTag
 }
+
+func MountBitBucketOAuthConfig(deployContext *deploy.DeployContext, deployment *appsv1.Deployment) error {
+	// mount BitBucket configuration
+	secrets, err := deploy.GetSecrets(deployContext, map[string]string{
+		deploy.KubernetesPartOfLabelKey:    deploy.CheEclipseOrg,
+		deploy.KubernetesComponentLabelKey: deploy.OAuthScmConfiguration,
+	}, map[string]string{
+		deploy.CheEclipseOrgOAuthScmServer: "bitbucket",
+	})
+
+	if err != nil {
+		return err
+	} else if len(secrets) == 1 {
+		// mount secrets
+		deployment.Spec.Template.Spec.Volumes = append(deployment.Spec.Template.Spec.Volumes,
+			corev1.Volume{
+				Name: secrets[0].Name,
+				VolumeSource: corev1.VolumeSource{
+					Secret: &corev1.SecretVolumeSource{
+						SecretName: secrets[0].Name,
+					},
+				},
+			})
+		deployment.Spec.Template.Spec.Containers[0].VolumeMounts = append(deployment.Spec.Template.Spec.Containers[0].VolumeMounts,
```

Comment:
We can move deployment.Spec.Template.Spec.Containers[0] to the local variable with name 'container' or something like that.

Reply:
`container := &deployment.Spec.Template.Spec.Containers[0]`
sounds good